luxonis_ml/data/parsers/luxonis_parser.py

```diff
@@ -111,7 +111,7 @@ def __init__(
         dataset_name = dataset_name or name.replace(" ", "_").split(".")[0]
 
         self.dataset = self.dataset_constructor(dataset_name=dataset_name, **kwargs)
-        self.parser = self.parsers[self.dataset_type](self.dataset)
+        self.parser = self.parsers[self.dataset_type](self.dataset, task_mapping or {})
```

Comment:
[Minor] Would it make sense to define the parameter as `task_mapping: Dict[LabelType, str] = {}` instead of this `or` statement?

Reply:
Python can't handle well mutable default parameters (although here it probably wouldn't be an issue but having `None` there is still always the safer option).